e2etest/bgptests/bgp.go

```diff
@@ -67,6 +67,9 @@ var _ = ginkgo.Describe("BGP", func() {
 			speakerPods, err := metallb.SpeakerPods(cs)
 			framework.ExpectNoError(err)
 			for _, pod := range speakerPods {
+				if len(pod.Spec.Containers) == 1 { // we dump only in case of frr
+					continue
```

Comment:
Mixed BGP backends are not supported. We could exit (break) the for-loop on the first speaker pod if its number of containers is equal to 1 (native mode).